pkgs/servers/blockbook/default.nix

```diff
@@ -12,25 +12,31 @@
 , lz4
 }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "blockbook";
-  version = "0.3.2";
-
-  goPackagePath = "blockbook";
+  version = "0.3.3";
+  commit = "b6961ca";
 
   src = fetchFromGitHub {
     owner = "trezor";
     repo = "blockbook";
     rev = "v${version}";
-    sha256 = "0hcgz4b7k8ia4dnjg6bbii95sqg3clc40ybwwc4qz3jv21ikc54x";
+    sha256 = "01nb4if2dix2h95xvqvafil325jjw2a4v1izb9mad0cjqcf8rk6n";
   };
 
-  goDeps = ./deps.nix;
+  modSha256 = "1zp06mpkxaxykw8pr679fg9dd7039qj13j5lknxp7hr8dga0jvpy";
 
   buildInputs = [ bzip2 zlib snappy zeromq lz4 ];
 
   nativeBuildInputs = [ pkg-config packr ];
 
+  buildFlagsArray = ''
+    -ldflags=
+       -X github.com/trezor/blockbook/common.version=${version}
+       -X github.com/trezor/blockbook/common.gitcommit=${commit}
```

Comment:
If you plan to package only stable versions, maybe we can set `common.gitcommit=unknown` and make the whole update process easier (we could drop the `commit` on line 18).

Reply:
Yes we could, but the reason I decided not to drop it is this:
![image](https://user-images.githubusercontent.com/791309/81622297-e57f9800-93f0-11ea-87d0-4711ae310023.png)

It results in a link which points straight to upstream, to that very commit, which I thought is kinda neat.

Reply:
Let's keep it that way then. It looks nice.